float32 start_x
float32 start_y
float32 goal_x
float32 goal_y
---
uint32[] data